### PR TITLE
Fix user count

### PR DESCRIPTION
**Pull request by rijkvanzanten**
- Users that had a direct policy attachment setting their app/admin status were not filtered out against their status causing non-active users to be counted. 
- Users with a direct app-policy attachment with an admin status through a role were double counted as app users

### CODE REVIEW

**Review by hanneskuettner — COMMENTED** [review]
I think these changes make sense, but I worked an them with you, so it's up to @licitdev to do the final review.

**Review by licitdev — APPROVED** [review]
Is a changeset needed for this PR?

**Comment by hanneskuettner** [comment]
> Is a changeset needed for this PR?

Good call, I think so.

File: .changeset/fuzzy-news-drop.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@directus/api': patch
+---
+
+Fixed user counting where users were double counted and inactive users with policies where counted as well
```
Comment (licitdev) [inline .changeset/fuzzy-news-drop.md:5]:
```suggestion
Fixed user counting where users were double counted and inactive users with policies were counted as well
```

**Pull request merged**